.env.example

```diff
@@ -24,6 +24,11 @@ NODE_ENV=development
 # FROM_EMAIL=
 # REPLY_TO_EMAIL=
 
+# OPTIONAL VARIABLES
+WHITELISTED_EMAILS="matt@gmail\.com|jane@yahoo\.com"
+# Description: This environment variable defines a regex pattern for allowed email addresses. 
+# Only emails that match this pattern will be able to sign up or log in.
+
```

Comment:
Please have a look further up the file, the Optional Variables section already exists.

The `WHITELISTED_EMAILS` line would have to be commented out as it's optional.

The description is good, but would have to go above the env var, not below. You can also drop the `Description:` prefix.